### ISSUE TITLE
Dictionary?

### ISSUE THREAD

**Issue by robjens** [spencermountain/nlp_compromise#56]
Are you using a own internal dictionary / algorithms to do the transformations etc? If so, and I seem to think this is the case, there is something off with the `conjugation` of the verb `"load"`:

```
{ infinitive: 'loa',
  present: 'loads',
  past: 'loaded',
  gerund: 'loading',
  doer: 'loaer',
  future: 'will loa' }
```
Now if I try something else, like `"to load"`:

```
{ infinitive: 'to load',
  present: 'to loads',
  past: 'to loaded',
  gerund: 'to loading',
  doer: 'to loader',
  future: 'will to load' }
```

This doesn't seem right either. Am I doing something wrong with entry of the string word(s) - some form I am missing? Or is this a corner case in the algorithm perhaps? Thought I'd at least report it :package: 

Otherwise, great solution: much thanks!

**Comment by spencermountain** [comment]
ah! eww! 
good find rob. I'll make a fix.
thanks man

**Comment by spencermountain** [comment]
hey, fixed in 1.1.3, just added it to the list of irregulars
thanks
